### DISCUSSION TITLE
Install bertopic has failed

### DISCUSSION THREAD

**Discussion by j-jsmith** [MaartenGr/BERTopic#967] [General]
I just tried to install bertopic on Python (version 3.11, using Pycharm, I have a Mac) and got an error message about conflicting dependencies (see attachement). It mentioned the package torch which is only compatible with version 3.7), do I need to install pyenv so that it works? Has anyone else had this problem? Can anyone explain to me what I need to do so I can install bertopic? I'm new to Python and have very little knowledge so far. 

<img width="1198" alt="Error message" src="https://user-images.githubusercontent.com/123753926/215123429-f76bf346-1858-4087-8296-e2cca7feef0b.png">

**Comment by MaartenGr** [comment]
Often, there is a little bit more to the error message which should explain what is exactly happening here. Having said that, I would expect that there might be some sub-dependency that does not yet work with python 3.11. My advice is to create a completely clean environment with python 3.8 and install BERTopic there. That should give you the least amount of issues with respect to python versions.

**Reply by j-jsmith** [reply]
Thank you so much, it works now!

**Comment by j-jsmith** [comment]
I have been following MaartenGr's Quick Start guide, however I have only been able to come as far as "topics, probs = topic_model.fit_transform(docs)". The code has been running for 10 minutes and nothing has happened. 
Does anyone know why?
Thank you for any help in advance! 


![Running for 10 minutes](https://user-images.githubusercontent.com/123753926/217573420-0638dc10-d6dc-4d44-a19d-c35a8a8e0aed.png)

**Reply by j-jsmith** [reply]
I found some more code by MaartenGr and this seems to work. Does anyone know what the difference is?
![This seems to work](https://user-images.githubusercontent.com/123753926/217581582-50ae2dc0-c856-4cbe-aa76-cf304d19357a.png)

**Reply by MaartenGr** [reply]
The difference here is that you used `verbose=True`. That parameters allows you to see what is happening when the model is currently running. However, the code in both images that you shared are doing the exact same thing. The only difference is that one is proving some logging and the other does not.
